geospaas/base_viewer/templates/base_viewer/elements.html

```diff
@@ -15,7 +15,13 @@
         <td class="place_ds">
             <div class="geometry_ref" ajax_url="{% url 'base_viewer:geometry_geojson' ds.geographic_location.pk %}">
             {% for url in ds.dataseturi_set.all %}
-            {{ url.uri}} <br>
+            {% if show_local %}
+              {{ url.uri}} <br>
+            {% else %}
+              {% if url.service != local_file_service %}
+                {{ url.uri}} <br>
+              {% endif %}
+            {% endif %}
```

Comment:
It would probably be best to put a placeholder showing that the path is hidden rather that not displaying the URL at all.
Otherwise it could be confusing in cases where datasets only have a local path.

Reply:
Done in the next commit.